src/Microsoft.VisualStudio.Composition/Configuration/NetFxAdapters.cs

```diff
@@ -13,9 +13,12 @@ namespace Microsoft.VisualStudio.Composition
     using System.Reflection;
     using System.Text;
     using System.Threading.Tasks;
+    using MessagePack;
+    using Microsoft.VisualStudio.Composition.Formatter;
     using Microsoft.VisualStudio.Composition.Reflection;
     using MefV1 = System.ComponentModel.Composition;
 
+    [MessagePackObject]
```

Comment:
This is a `static` class. As such, it cannot be serialized because there is never an object for it.